lms/djangoapps/instructor/views/instructor_dashboard.py

```diff
@@ -328,6 +328,7 @@ def _section_membership(course, access):
         'section_key': 'membership',
         'section_display_name': _('Membership'),
         'access': access,
+        'ccx_is_enabled': course.enable_ccx if settings.FEATURES.get('CUSTOM_COURSES_EDX', False) else False,
```

Comment:
I have a bit of a preference to do the logic outside of this definition, to make the logic clearer and easier to change. Eg,
```python
ccx_enabled = False
if settings.FEATURES.get('CUSTOM_COURSES_EDX', False) and course.enable_ccx:
    ccx_enabled = True
section_data = {
    ...
    'ccx_is_enabled': ccx_enabled,
    ...
}
```